Updated documentation for our ADE development webpage:

srcarter3 opened the pull request:
-added notes about running as root when applicable
-added a note to check for libGLU package (because this was an issue I encountered when creating a ADE VM from scratch)
-alphabetized our git clone list and added the goesr repo
-added note about switching branches if desired, before importing into eclipse
-updated the command for running eclipse (no longer has '.sh' on executable)
-Reformated section 5 to have subsections/subheaders
-made paths more specific in section 5 so there is no confusion on where settings are to be found
-added screenshots for importing git repos and cleaning workspace
-reformated section 6 to have subsections/headers
-added screenshots for clarity

Commit:
Updated documentation for our ADE development webpage:
-added notes about running as root when applicable
-added a note to check for libGLU package (because this was an issue I encountered when creating a ADE VM from scratch)
-alphabetized our git clone list and added the goesr repo
-added note about switching branches if desired, before importing into eclipse
-updated the command for running eclipse (no longer has '.sh' on executable)
-Reformated section 5 to have subsections/subheaders
-made paths more specific in section 5 so there is no confusion on where settings are to be found
-added screenshots for importing git repos and cleaning workspace
-reformated section 6 to have subsections/headers
-added screenshots for clarity

tiffanycmeyer13 commented on docs/dev/awips-development-environment.md:
```diff
@@ -41,11 +43,15 @@ proxy=_none_
 
 ## 3. Install the ADE
 
+!!! note "The following may need to be run as *root* user."
```
For some reason this note isn't showing up with the correct box (but the one above is...)

![image](https://user-images.githubusercontent.com/65614396/119003164-20e16a80-b953-11eb-9b08-4058d1fb36b2.png)

tiffanycmeyer13 commented on docs/dev/awips-development-environment.md:
```diff
@@ -57,69 +63,102 @@ If it's not already installed, install git:
+  ![git import](../images/gitImport1.png)
+  
+1. Continue with the default selection, Existing local repository > **Add..** > add each of the git repos (for example `.../awips2-core`) > check the checkbox > **Finish**
+  ![add git repo](../images/gitImport2.png)
```
Your path in the image is /home/awips/awipsRepos/**_unusedRepos_**/awips2-core

Might change that path

Commit:
Small changes to the ADE webpage based on feedback from Tiffany:
- changed all '>' style notes to the '!!!' style
- moved the "run as root" message to the top of the page and removed it from each subsection
- added 'sudo vi /etc/yum.repos.d/awips2.repo' command in the second section, for clarity
- updated three images: gitImport2 - changed the path, runApplication % debugApplication - extended the view so you can see the project in the project explorer

Commit:
Fix a path on one of the gitImport images
